components/brave_wallet/renderer/brave_wallet_js_handler.cc

```diff
@@ -26,6 +26,8 @@
 #include "third_party/blink/public/web/web_script_source.h"
 #include "ui/base/resource/resource_bundle.h"
 
+#include "brave/components/brave_wallet/renderer/eth_request_parser.h"
```

Comment:
move this one after `#include "brave/components/brave_wallet/renderer/brave_wallet_response_helpers.h"`

Reply:
Fixed in d8c9c09e62fadd786f09bfdf932a2573f3dd5358